src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx

```diff
@@ -144,10 +176,20 @@ const IntegrationTestSection: React.FC<Props> = ({ isInPage, edit }) => {
         <InputField
           name="integrationTest.path"
           label="Path in repository"
+          isDisabled={edit}
           helpText="Subdirectory for the application source code."
           data-test="git-path-repo"
           required
         />
+        <DropdownField
+          name={'integrationTest.environmentName'}
+          label="Environment"
+          placeholder={ITSEnvName ?? ENVIRONMENTS.DEFAULT}
+          onChange={setEnvironment}
+          items={dropdownItems}
+          helpText="Select an environment for your test to run on. It can not be changed later"
```

Comment:
I think that it should be possible to change the base environment after the fact.
The text should also probably need to make clear that the environment that's used for testing will be an ephemeral copy of the selected environment, something like this (but feel free to refine it further):
```
Select an existing environment that the ephemeral test environment for this test  will be based on.
```

Reply:
I copied the text from the UX copy. cc: @MariaLeonova @beaumorley input here pls

Reply:
"Select an existing environment for us to make a copy of. We'll use the copy as a test environment and then delete it." How about this? Tried to make it sound a little more conversational @dirgim @abhinandan13jan

Reply:
If you can change the environment later, we should update the UX copy. What about something like this?

_Select a static environment for your test to run on. This will be used for the ephemeral test environment._

Reply:
Thanks for calling that out, Kruno. I had a similar comment in a Miro for these. I think the suggestions here are on the right track. It doesn't all have to go directly into the UI, some detail could be reserved for the docs, but other aspects here I could see being helpful to some folks is clarifying what it means "to copy" (config and env variables will be reused and will match the state of the selected env at the time it is copied), this will be created and deleted each time this test is invoked, an environment doesn't have to be used, they can select no environment (how do they know when they can skip it?).

Reply:
@wolfeallison @beaumorley updated